docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -175,6 +175,30 @@ Type: UInt64
 
 Default: `512`
 
+## backups
+
+Settings for backups, used when writing `BACKUP TO File()`.
+
+:::note
+[ClickHouse Cloud](https://clickhouse.com/cloud) provides automatic backups to object storage.
+:::
+
+The following parameters can be configured:
+
+| Parameter                           | Description                                                                                                                                                                                |
+|-------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `allowed_path`                      | Path to backup to when using `File()`. This parameter must be set in order to use `File`.                                                                                                  |
```

Comment:
Should specify where the path is relative to by default. I see usage of both `backups` and `/backups`